rdr_service/model/participant_summary.py

```diff
@@ -446,6 +446,29 @@ def enrollmentSiteId(cls):
     UTC timestamp indicating the latest time RDR was aware of signed and uploaded EHR documents
     """
 
+    # DA-3156:  Separate tracking for participant-mediated EHR (e.g., from CE participants)
+    # These values will be backfilled based on a report of mediated EHR activity imported into a temp table
+    wasParticipantMediatedEhrAvailable = Column(
+        "was_participant_mediated_ehr_available",
+        Boolean,
+        nullable=False,
+        server_default=expression.false()
+    )
+    """
+    A true or false value that indicates whether participant-mediated Electronic Health Records (EHR)
+    have ever been present for the participant.  Mediated EHR is a separate EHR source than HPO-provided EHR
+    """
+
+    firstParticipantMediatedEhrReceiptTime = None
+    """
+    UTC timestamp indicating first reported occurrence of participant-mediated EHR content
+    """
+
+    latestParticipantMediatedEhrReceiptTime = None
+    """
+    UTC timestamp indicating the latest reported occurrence of participant-mediated EHR content
+    """
```

Comment:
Should `firstParticipantMediatedEhrReceiptTime` and `latestParticipantMediatedEhrReceiptTime` load the values from the table?

Reply:
Can you clarify?  This PR is just trying to establish the new (but as yet unpopulated fields) in the model/schema.  A future script/backfill will apply values to the fields for affected participants based on a one-time file ingestion (via a scripted tool or backfill that gets the values from a temp table).

Reply:
Yep, that's what I meant. Sorry it wasn't clear.